README.md

```diff
@@ -78,15 +78,25 @@ XAMPP is a very easy to install Apache Distribution for Linux, Solaris, Windows
 
 This [video](https://youtu.be/Yzksa_WjnY0) walks you through the installation process for Windows but it should be similar for other OSs.
 
-### Config File
+### Docker
 
-DVWA ships with a dummy copy of its config file which you will need to copy into place and then make the appropriate changes. On Linux, assuming you are in the DVWA directory, this can be done as follows:
+It is possible to run DVWA with containers.
 
-`cp config/config.inc.php.dist config/config.inc.php`
+Prerequisites: Docker and Docker Compose.
 
-On Windows, this can be a bit harder if you are hiding file extensions, if you are unsure about this, this blog post explains more about it:
+- If you are using Docker Desktop, both of these should be already installed.
+- If you prefer Docker Engine on Linux, make sure to follow their [installation guide](https://docs.docker.com/engine/install/#server).
+**Tip:** You might want to run Docker without superuser privileges.
+Follow [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) for more information.
 
-[How to Make Windows Show File Extensions](https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/)
+Then, to get started:
+
+1. Run `docker version` and `docker compose version` to see if you have Docker and Docker Compose properly installed.
```

Comment:
Tell them what they are looking for if they run correctly. People will type the command and then move on, they don't always read the error messages, even if they say "command not found".